### DISCUSSION TITLE
Unable to run SSCHA in cluster through sscha.cluster module

### DISCUSSION THREAD

**Discussion by Kalith-Ismaik** [SSCHAcode/python-sscha#135] [Q&A] [answered]
I am trying to use make some quantum espresso calculations through SSCHA (sscha.cluster), but right after the execution of jupyter node to execute the line of code of "ensemble.compute_ensemble(espresso_calc, cluster = my_hpc)", it is giving of the response of "Error while writing input file ESP_0" for every configurations and nor creaating any input file neither uploading any .pwi file in to cluster for calculations.
It would be really helpful if someone could guide me overcome this issue.
[Tutorial_PbTe.pdf](https://github.com/SSCHAcode/python-sscha/files/11848028/Tutorial_PbTe.pdf)

**Comment by mesonepigreco** [comment, marked as answer]
Hi, sorry for the delay in answering, last week we were very buisy with the SSCHA school.
We now have a new tutorial for the cluster (and a whole set of tutorials for the SSCHA).
You can find them at this link:
[SSCHA Tutorials](http://sscha.eu/Schools/2023/sscha-school-2023.pdf)

In general, are you using the last version of the code? (you can update it with pip install python-sscha cellconstructor -U)

The error while writing input file ESP_0 means that the code is not able to generate the input files for quantum espresso and write them on the hard disk. Maybe there is some error on the calculator side? Try to follow the latest tutorial I link for the Cluster part (section 3.2). 

If you continue to have issues, could you report the code explicitly that is failing with more details about the error message?
